### PR TITLE
Enhancement for block summarizer

### PR DESCRIPTION
**Pull request by DevAseel**
The following PR includes the following: 
- Made analytics happen for pairs within the current block height which allowed us to reduce the amount of saved data in block summary table from 1,365,381 to 1,258,876 across the dump file data. 
- Ran the block summarizer service across the dump file data.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -13,8 +13,6 @@ READ_DB_CONNECTION_USERNAME = postgres
 READ_DB_CONNECTION_PASSWORD = casper_db_password
 READ_DB_CONNECTION_PORT = 5432
 
-# if this variable is set, then the hourly_summarizer will re-do summarization for the time range FORCE_RECALCULATE_START_HOUR TO FORCE_RECALCULATE_START_HOUR + 1 hour
```
Comment (specialistvlad) [inline .env.example:16]: @DevAseel should we remove comment?

Comment (DevAseel) [reply .env.example:16]: I think we should keep it, including @MessaoudAzidane since I believe he knows best.

Comment (MessaoudAzidane) [reply .env.example:16]: I prefer to keep it please.

**Review by mrrfid — APPROVED** [review]
There are a few typos in the comments; "records" is spelled incorrectly, but this is minor and will get corrected in a future revision. Great job!

**Pull request merged**